website/docs/r/app_configuration.html.markdown

```diff
@@ -220,6 +235,14 @@ An `identity` block exports the following:
 
 ---
 
+A `replica` block exports the following:
+
+* `id` - The replica ID.
+
+* `endpoint` - The URL of the replica.
```

Comment:
Please replicate this, i.e. phrasing and casing to the other doc changes including the data source docs
```suggestion
* `endpoint` - The URL of the App Configuration Replica.
```

Reply:
fixed, thanks.